project/.travis.yml.twig

```diff
@@ -64,6 +63,8 @@ matrix:
 {% endfor %}
     - php: '{{ php|last }}'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: hhvm
+      dist: trusty
   allow_failures:
     - php: nightly
     - php: hhvm
```

Comment:
the allow_failures just lists which builds are allowed to fail. the build remains defined above. it was defined in the `php:` section and now moved into the matrix part so that we can specify additional flags along with the php=hhvm version.